cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/ConversionTester.java

```diff
@@ -230,7 +227,7 @@ private static ValidationConfig loadFlowValidationConfig(FileSystem fs, double t
         InMemoryPlatformConfig pconfig = new InMemoryPlatformConfig(fs);
         pconfig
                 .createModuleConfig("componentDefaultConfig")
-                .setStringProperty("LoadFlowFactory", LoadFlowFactoryMock.class.getCanonicalName());
+                .setStringProperty("LoadFlowFactory", "LoadFlowMock");
```

Comment:
To be consistent, you should change the name of this property to "load-flow-name"? If you do this change, don't forget to update all the references to componentDefaultConfig in the website

Reply:
I remove this unused componentDefaultConfig config because load-flow-name config is now done in validation specific config.